pydicom/util/codify.py

```diff
@@ -76,26 +74,39 @@ def code_imports():
     return line_term.join((line1, line2, line3))
 
 
-def code_dataelem(dataelem,
-                  dataset_name="ds",
-                  exclude_size=None,
-                  include_private=False):
+def code_dataelem(
+    dataelem: DataElement,
+    dataset_name: str = "ds",
+    exclude_size: Optional[int] = None,
+    include_private: bool = False
+) -> str:
     """Code lines for a single DICOM data element
 
-    :arg dataelem: the DataElement instance to turn into code
-    :arg dataset_name: variable name of the Dataset containing dataelem
-    :arg exclude_size: if specified, values longer than this (in bytes)
-                       will only have a commented string for a value,
-                       causing a syntax error when the code is run,
-                       and thus prompting the user to remove or fix that line.
-    :return: a string containing code to recreate the data element
-             If the data element is a sequence, calls code_sequence
+    Parameters
+    ----------
+
+    dataelem : DataElement
+        The DataElement instance to turn into code
+    dataset_name : str
+        The variable name of the Dataset containing dataelem
+    exclude_size : Union[int, None]
+        If specified, values longer than this (in bytes)
+        will only have a commented string for a value,
+        causing a syntax error when the code is run,
+        and thus prompting the user to remove or fix that line.
+
+    Returns
+    -------
+    code: str
```

Comment:
Is this name some convenience? We usually don't write a name for the return value, and I find it a bit confusing (because the name does not actually exist in the code). Same in all docstrings below.

Reply:
I thought there was a name there, but searching code I see you are right - will update those,